src/docopt/dispatch.nim

```diff
@@ -1,6 +1,9 @@
 import docopt, macros, strutils, sequtils, typetraits
 
 macro runUserImplemented(x: typed, fallback: untyped): untyped =
+  ## This macro checks takes a type from a generic and checks if there exists
```

Comment:
First "checks" is redundant (probably changed your mind mid sentence what to write.